scss/_variables.scss

```
@@ -620,9 +620,8 @@ $blockquote-footer-font-size: $small-font-size !default;
 
 $hr-margin-y:                 $spacer !default;
 $hr-color:                    inherit !default;
-$hr-bg-color:                 currentColor !default;
-$hr-border-width:             0 !default;
-$hr-height:                   $border-width !default;
+$hr-border-color:             null !default;
+$hr-border-width:             1px !default;
```

To avoid this being a breaking change, we need to keep the old variables around, even if they're `null`. I've also restored the re-use of `$border-width` in the new `$hr-border-width` variable.

```suggestion
// fusv-disable
$hr-bg-color: null !default; // Deprecated in v5.2.0
$hr-height: null !default; // Deprecated in v5.2.0
// fusv-enable

$hr-border-color: null !default; // Allows for inherited colors
$hr-border-width: $border-width !default;
```